db/queries/cards.sql

```diff
@@ -26,7 +26,11 @@ WITH matching_cards AS (
   WHERE user_id = $1
     AND lexicon_name = $2
     AND next_scheduled <= $3
-  ORDER BY next_scheduled ASC
+  ORDER BY
+    -- When short-term scheduling is enabled, we want to prioritize review cards
+    -- over all other states (`1` === Review in the FSRS card state)
```

Comment:
Will this actually work? See here:

https://github.com/open-spaced-repetition/go-fsrs/blob/main/scheduler_basic.go#L34

It seems for the short-term scheduler that the card gets placed in a "Learning" state, if it was in a New state to begin with. It only gets placed in a "Review" state if you mark it Easy.

Maybe we should just compare if the state is NOT New (but please verify this makes sense). I think cards can only be New once, when they are added, prior to asking them. These are the ones that should be prioritized least, I think.